docs/components_and_bundles/bundles/SyliusResourceBundle/create_resource.rst

```diff
@@ -87,7 +87,7 @@ Below you can see the usage for specifying custom options, in this case, ``valid
             _controller: app.controller.book:createAction
             _sylius:
                 form:
-                    type: app_book_custom
+                    type: App\Form\BookType
```

Comment:
Shouldn't it be `App\Form\Type\BookType` everywhere?

Reply:
According to line 69(https://github.com/Sylius/Sylius/pull/11051/files#diff-bf33602b03f5b0c9092fd45352f79241R69) apparently no, but I would say yes